internal/service/cloudsearch/domain.go

```diff
@@ -1146,6 +1182,10 @@ func flattenIndexFieldStatus(apiObject *cloudsearch.IndexFieldStatus) (map[strin
 			tfMap["sort"] = aws.BoolValue(v)
 		}
 
+		// Defaults not returned via the API.
```

Comment:
For today. Then it will be returned, and then it won't. Seems like large scale we should have a better way to handle this - consistent, small changes in what the API returns.

Reply:
It's also the interaction between DiffSuppress and the Set's hash function.

Reply:
Also to be able to have a calculated `Default` value.

Reply:
For even just the simple case of a string. e.g., If you send nothing, the API will return the default value. Then maybe a few months later it will return an empty string, but set the default. Then a while later it will return nil. If we sent nothing, and receive back any of 1) the default, 2) an empty string, or 3) nil/null, it should be the same (unless it shouldn't be).